fdbserver/TenantBalancer.actor.cpp

```diff
@@ -350,18 +373,62 @@ struct TenantBalancer {
 
 	Future<Void> recover() { return recoverImpl(this); }
 
+	Optional<Key> getSrcPrefix(std::string destTagName) const {
+		auto itr = destTagNameToSrcPrefix.find(destTagName);
+		if (itr == destTagNameToSrcPrefix.end()) {
+			return Optional<Key>();
+		}
+		return itr->second;
+	}
+
+	Future<bool> isTenantEmpty(Reference<ReadYourWritesTransaction> tr, Database db, Key prefix) {
```

Comment:
This parameter isn't used. Also, this function has a wait, so it needs to be made into an actor.

```suggestion
	Future<bool> isTenantEmpty(Reference<ReadYourWritesTransaction> tr, Key prefix) {
```